src/components/Tracks/Track.js

```diff
@@ -1,24 +1,33 @@
-import {
-  ListItem,
-  ListItemButton,
-  Typography,
-} from "@mui/material";
+import { ListItem, ListItemButton, Typography } from "@mui/material";
 import React from "react";
+import { useCallback } from "react";
+import { useCurrentTrackDispatch } from "../../store/track-context";
+
+function Track({ track, selected }) {
+  const setCurrentTrack = useCurrentTrackDispatch();
```

Comment:
bad naming...

You really don't need to know the current track?

Reply:
BTW, why do you need the hook it here if you can have `onClick` event as property?